CHANGELOG.next.asciidoc

```diff
@@ -141,6 +141,94 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- Add AWS SQS metricset. {pull}10684[10684] {issue}10053[10053]
```

Comment:
These extra changelog entries need to be removed.

Reply:
that was fast, just updated changelog